ontology/yaml/resources/HVAC/entity_types/MAU.yaml

```diff
@@ -16,6 +16,21 @@
 ### Canonical Types  ###
 ########################
 
+MAU_STC_SFM_SFVSC_FDPM_FDPSM_CREFM_SDM_SSPC_HWVM:
+  description: "Make-up air unit with heating control using wet coil to maintain air flowrate."
```

Comment:
Thanks for the clarification - I suggest using `HWSC` (which combines the functionality of STC and HWVM into one, so you can remove both of those types) because the heating valve + temperature control are combined functionality and should be modeled as such.

Reply:
Great spot, thank you. I have recommitted